erpnext/schools/doctype/fees/fees.json

```diff
@@ -81,19 +142,18 @@
    "bold": 0, 
    "collapsible": 0, 
    "columns": 0, 
-   "fieldname": "academic_term", 
-   "fieldtype": "Link", 
+   "fieldname": "include_payment", 
```

Comment:
This will be used to send the payment request if we are creating the fee records from the fee schedule.